### ISSUE TITLE
[Feat]: Pre-trained model with fewer parameters (such as 0.7M or 0.5M)

### ISSUE THREAD

**Issue by flwang1998** [microsoft/mattersim#91]
### Contact Details

fl0558@foxmail.com

### Feature Description

Currently, the pre-trained models have a large number of parameters, which is slow for running large-scale molecular dynamics simulations. Are there pre-trained models with fewer parameters available?

### Motivation

1. Small-parameter models are very useful in large-scale MD, which would expand the influence of MatterSim. 
2. Although small parameters may reduce the accuracy of pre-trained models, the precision may be improved through fine-tuning.

Thank you for considering this issue.

Best wishes.

### Contribution Interest

- [x] I'm interested in potentially implementing this feature
- [ ] I can provide guidance, but cannot implement
- [x] I'm just suggesting the feature for the community

### Code of Conduct

- [x] I agree to follow the project's Code of Conduct

**Comment by yanghan234** [comment]
Thanks for your interest in MatterSim. Currently we don't have smaller public version of MatterSim. Please stay tuned for potential updates in this repo in the future!
